Cargo.toml

```diff
@@ -5,7 +5,7 @@ repository = "https://github.com/softdevteam/vob/"
 version = "1.0.0"
 authors = ["Laurence Tratt <laurie@tratt.net>"]
 readme = "README.md"
-license = "Apache-2.0 OR MIT OR UPL-1.0"
```

Comment:
Unlucky :) How about a comment to the effect of the PR description?

Reply:
Good idea.